resources/public/js/externs.js

```diff
@@ -134,3 +134,22 @@ CarrotGA.NULL_VALUE = {};
 CarrotGA.clientid = '';
 var init_fullstory = function(){};
 var FS = function(){};
+// Ziggeo
+var ZiggeoApi = function(){};
+ZiggeoApi.Events = {};
+ZiggeoApi.Events.on = function(){};
+ZiggeoApi.Events.off = function(){};
+ZiggeoApi.token = {};
+ZiggeoApi.Config = {};
+ZiggeoApi.Config.webrtc = {};
+ZiggeoApi.V2 = {};
+ZiggeoApi.V2.Application = function(){};
+ZiggeoApi.V2.Recorder = function(){};
+ZiggeoApi.V2.Recorder.activate = function(){};
+ZiggeoApi.V2.Recorder.destroy = function(){};
+ZiggeoApi.V2.Recorder.get = function(){};
+ZiggeoApi.V2 = {};
```

Comment:
Doesn't this overwrite the previous definitions?

Reply:
Good catch, i fixed it but i don't think it was a problem since it was working.